mrp_subcontracting_purchase/README.rst

```diff
@@ -0,0 +1,2 @@
+This module is a backport from Odoo SA and as such, it is not included in the OCA CLA.
```

Comment:
```suggestion
This bridge module adds some smart buttons between Purchase and Subcontracting

**DISCLAIMER:** This module is a backport from Odoo SA and as such, it is not included in the OCA CLA.
```

Reply:
Oh, I see that you have a DESCRIPTION.rst in the `readme` folder, so you have to introduce the disclaimer there.

Reply:
> Oh, I see that you have a DESCRIPTION.rst in the `readme` folder, so you have to introduce the disclaimer there.

Resolved